plugin/web/extensions/sidebar/core/components/hooks/projectHooks.ts

```diff
@@ -193,6 +193,17 @@ export default ({
     postMsg({ action: "removeAllDatasetsFromScene" });
   }, []);
 
+  const handleProjectDatasetsUpdate = useCallback((datasets: DataCatalogItem[]) => {
+    updateProject(({ sceneOverrides }) => {
+      const updatedProject = {
+        sceneOverrides,
+        datasets,
+      };
+      postMsg({ action: "updateProject", payload: updatedProject });
+      return updatedProject;
+    });
+  }, []);
```

Comment:
This isn't necessary since we have the handleDatasetUpdate function in the datasetHooks.
BTW, this handleProjectDatasetUpdate is problematic because it only updates the project that is used for UI. It is missing updating overrides and other things that are necessary when a dataset is updated